lte/gateway/python/magma/kernsnoopd/tests/packet_counter_tests.py

```diff
@@ -0,0 +1,125 @@
+"""
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import unittest
+from socket import htons
+from unittest.mock import MagicMock
+
+from magma.kernsnoopd.handlers import PacketCounter
+
+
```

Comment:
Just commenting to say thank you for adding such well written unit tests ![](https://media2.giphy.com/media/i21tixUQEE7TEqwmYa/giphy.gif)